### DISCUSSION TITLE
Strava Heatmap Overlay doesn't work

### DISCUSSION THREAD

**Discussion by drazde** [osmandapp/OsmAnd#21389] [Maps Related]
I used to use Strava Heatmap like overlay, I found it very usefull to know if a path is frequentated by bike or for hiking and also for improve the path on OpenStreetmap.
Now OsmAnd doesn't download the tiles with the same url 
Why? 

The url that I use is:  _https://proxy.nakarte.me/https/heatmap-external-b.strava.com/tiles-auth/all/bluered/{z}/{x}/{y}.png?px=256_
(for all activities), the service seems work; I use it for mapping too.

I also try with other url but nothing

**Comment by Nine-Tailed** [comment]
I support the question, because in guru maps the strava heat map continues to work.

**Reply by mariush444** [reply]
try
https://strava-heatmap.tiles.freemap.sk/all/hot/{0}/{1}/{2}.png?px=256

**Reply by Nine-Tailed** [reply]
Thanks, I know such a link, but I preferred the nakarte.me proxy for the download speed.
And here it is more interesting why the osmand stopped downloading tiles, when other applications continue to work normally.

**Reply by drazde** [reply]
> try
> https://strava-heatmap.tiles.freemap.sk/all/hot/{0}/{1}/{2}.png?px=256

I tryed but seem doesn't work

**Reply by mariush444** [reply]
AFAIK Strava stoped sharing their data as previous time. I can't find  official info about that now but there was it as I remember. 
Strange that others apps can still use it if thet don't use free map. Why freemap still works is strange also.
Generally it is not problem with osmand.

**Reply by Nine-Tailed** [reply]
> I tryed but seem doesn't work

The format should not be sqlitedb. For some reason it doesn't work with it.

Or use this link by installing the map source.
Proxy freemap.sk
https://t.me/OsmAndMaps/52299

Below are links to proxies nakarte.me, I think I fixed them, I specified the user agent

https://t.me/ruosmand/167446
https://t.me/ruosmand/167447

**Reply by drazde** [reply]
Ok it works! Thanks @Nine-Tailed
